app/src/main/java/zapsolutions/zap/setup/ConnectRemoteNodeActivity.java

```diff
@@ -314,4 +350,48 @@ public void onRequestPermissionsResult(int requestCode,
         }
     }
 
+    private void showWalletNameInput(RemoteConfiguration remoteConfiguration) {
```

Comment:
I'm wondering if we can somehow extract this and only have it at one place since its almost the same logic.
https://github.com/LN-Zap/zap-android/pull/64/files#diff-5977ac2d4d41ccaa95fc1bb9fa1ff18fR101